### PR TITLE
fix: Revert chunkFileNames to include hash

### PR DESCRIPTION
**Pull request by juliushaertl**
Otherwise the browser cache is not refetching updated files.

I noticed after the upgrade on our instance today that the files app was having weired styles, only a force refresh helped. Not sure this has been exactly this issue, but as I talked with someone before (likely @susnux) I thought I'd open that PR.

### CODE REVIEW

**Comment by susnux** [comment]
Pro of this:
- Simple
- built-in cache buster

Con of this:
- Much noise on the repo ("deleted foo-123.mjs" "added foo-456.mjs")

---

I can live with both and I fear that simply patching `?v=hash` to the imports (really easy vite plugin) would not work as expected as we also need to patch the preload stuff...

**Review by susnux — APPROVED** [review]

File: lib/appConfig.ts [hunk 1]
```diff
@@ -221,7 +221,7 @@ export const createAppConfig = (entries: { [entryAlias: string]: string }, optio
 								return `js/${assetsPrefix}[name].mjs`
 							},
 							chunkFileNames: () => {
-								return 'js/[name].chunk.mjs'
+								return 'js/[name]-[hash].mjs'
```
Comment (susnux) [inline lib/appConfig.ts:224]: We could also cleanup the directory (if someone cares) with:
```suggestion
								return 'js/chunks/[name]-[hash].mjs'
```

Comment (juliushaertl) [reply lib/appConfig.ts:224]: Makes sense, let me push that here as well

Comment (ShGKme) [reply lib/appConfig.ts]: Or `js/[name]-[hash].chunk.mjs` to be consistant with css chunks?

Comment (ShGKme) [reply lib/appConfig.ts]: btw, won't we have the same issue with CSS chunk?
https://github.com/nextcloud-libraries/nextcloud-vite-config/blob/main/lib/plugins/CSSEntryPoints.ts#L49

**Review by susnux — APPROVED** [review]

**Pull request merged**